examples/semi_supervised/plot_label_propagation_digits.py

```diff
@@ -45,40 +48,59 @@ class will be very good.
 
 unlabeled_set = indices[n_labeled_points:]
 
-# #############################################################################
+# %%
 # Shuffle everything around
 y_train = np.copy(y)
 y_train[unlabeled_set] = -1
 
-# #############################################################################
-# Learn with LabelSpreading
+# %%
+# Semi-supervised learning
+# ------------------------
+#
+# We fit a :class:`~sklearn.semi_supervised.LabelSpreading` and use it to predict
+# the unknown labels.
+from sklearn.semi_supervised import LabelSpreading
+from sklearn.metrics import classification_report
+
 lp_model = LabelSpreading(gamma=0.25, max_iter=20)
 lp_model.fit(X, y_train)
 predicted_labels = lp_model.transduction_[unlabeled_set]
 true_labels = y[unlabeled_set]
 
-cm = confusion_matrix(true_labels, predicted_labels, labels=lp_model.classes_)
-
 print(
     "Label Spreading model: %d labeled & %d unlabeled points (%d total)"
     % (n_labeled_points, n_total_samples - n_labeled_points, n_total_samples)
 )
 
-print(classification_report(true_labels, predicted_labels))
+# %% Classification report
+# ------------------------
+classification_report(true_labels, predicted_labels)
```

Comment:
We need to print it otherwise we get the repr of the string which escapes the special chars like end line.
```suggestion
print(classification_report(true_labels, predicted_labels))
```